code/modules/mob/living/simple_animal/hostile/mining_mobs/basilisk.dm

```diff
@@ -79,15 +79,14 @@
 	movement_type = FLYING
 	robust_searching = 1
 	crusher_loot = /obj/item/crusher_trophy/watcher_wing
+	crusher_drop_mod = 10
 	loot = list()
 	butcher_results = list(/obj/item/stack/ore/diamond = 2, /obj/item/stack/sheet/sinew = 2, /obj/item/stack/sheet/bone = 1)
 
 /mob/living/simple_animal/hostile/asteroid/basilisk/watcher/random/Initialize()
 	. = ..()
 	if(prob(1))
 		if(prob(75))
```

Comment:
remove this if statement, otherwise there's only a 0.75% chance it'll spawn, instead of a 1%

Reply:
where else does it dictate drop chance for that mod because i dont see it